### PR TITLE
CSS fix for transparent dropdown options. Fixes #26.

### LINKED ISSUE #26: Block Type Dropdown: <option> text is transparent
It looks like the text of the `option` elements inside of the Block Type dropdown is set to transparent, which has the affect of making the text invisible (at least in Chrome 49).

### PR DESCRIPTION
**Pull request by amcintyre-cs**

### CODE REVIEW

File: src/ui/Dropdown.css [hunk 1]
```diff
@@ -76,13 +76,19 @@
    default focus styles. This is better for accessibility. */
 @media screen and (-webkit-min-device-pixel-ratio:0) {
   .root select {
-    opacity: 1;
     color: transparent;
+    opacity: 1;
```
Comment (sstur) [inline src/ui/Dropdown.css]: Why did we move this line? Maybe it's not intentional? In the effort of keeping changes clean and minimal, best to only change the lines that need changing. This helps preserve diff blame.

**Comment by sstur** [comment]
Thanks @amcintyre-cs. I appreciate you finding this bug and submitting a fix!

File: src/ui/Dropdown.css (hunk 1 above)
Comment (amcintyre-cs) [inline src/ui/Dropdown.css]: I alphabetized it out of habit. I'll flip it back to where it was.

Comment (sstur) [inline src/ui/Dropdown.css]: Oh, that makes sense actually. I'm not particularly glued to the previous order, just like that the PR touch only the code it needs to fix.

Comment (amcintyre-cs) [inline src/ui/Dropdown.css]: That makes total sense to me.

**Comment by sstur** [comment]
Looks good to me. Thanks!

**Pull request merged**